Neos.ContentRepository/Classes/Migration/Command/NodeCommandController.php

```diff
@@ -123,7 +123,7 @@ public function migrateCommand($version, $confirmation = false, $direction = Mig
      * List available and applied migrations
      *
      * @return void
-     * @see neos.contentrepository.migration:node:migrate
+     * @see neos.contentrepository.migration:node:migrationstatus
```

Comment:
Why should this point to itself?

Reply:
In the [KickstartCommandController](https://github.com/neos/flow-development-collection/blob/8.3/Neos.Kickstarter/Classes/Command/KickstartCommandController.php) its done the same way. I thought this would be a hint. So i know what to look for in the neos documentation on readthedocs.

Reply:
… and here in reverse.